src/services/models/telemetryModels.js

```diff
@@ -23,10 +23,22 @@ export const toRuleModel = (response = {}) => {
     'action.type': 'type'
   });
   return update(model, {
-    severity: { $set: (model.severity || '').toLowerCase() }
+    severity: { $set: (model.severity || '').toLowerCase() },
+    actions: { $set: (response.Actions || []).map(toActionModel)}
```

Comment:
nit: Add space before the last `}`
`[]).map(toActionModel) }`

Reply:
fixed